src/view/ChewingEditor.cpp

```diff
@@ -223,8 +230,12 @@ void ChewingEditor::setupImport()
         model_, SIGNAL(importCompleted(bool, const QString&, size_t, size_t)),
         ui_.get()->notification, SLOT(notifyImportCompleted(bool, const QString&, size_t, size_t))
     );
+
+    // initialize import_count
+    import_count = 0;
```

Comment:
You should always hide information details as possible since it is object-oriented programming paradigm. When I said "it would be", the statement implies the feasibility and use case.